src/Tizen.Network.Nfc/Tizen.Network.Nfc/NfcNdefMessage.cs

```diff
@@ -67,7 +67,35 @@ public NfcNdefMessage()
 
         internal NfcNdefMessage(IntPtr messageHandle)
         {
+            if (messageHandle == IntPtr.Zero)
+            {
+                return;
+            }
+
             _messageHandle = messageHandle;
+
+            int recordCount;
+            int ret = Interop.Nfc.NdefMessage.GetRecordCount(_messageHandle, out recordCount);
+            if (ret != (int)NfcError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to GetRecordCount, Error - " + (NfcError)ret);
+                NfcErrorFactory.ThrowNfcException(ret);
+            }
+
+            for (int i = 0; i < recordCount; i++) {
+                IntPtr recordHandle;
+                ret = Interop.Nfc.NdefMessage.GetRecord(_messageHandle, i, out recordHandle);
```

Comment:
Do you know about ownership of `recordHandle`?
Who should release `recordHandle`?